docs/achievements.md

```diff
@@ -0,0 +1,135 @@
+# achievements.lua
+This module is responsible for tracking and saving achievement progress for your game. Can be used as a standalone library, or in congruence with [toasts.lua](toasts.md) or [viewer.lua](viewer.md)
+
+## Getting Started
+To import the main library, include `achievements.lua` in your project and import it. This will create a global variable named `achievements` containing the module.
+
+```lua
+-- while the game is setting up
+import "./path/to/achievements" -- don't include ".lua" in your import statement
+achievements.initialize(data) -- initialize your data blob prior during your game setup
+
+...
+
+-- while the game is running
+achievements.grant("my_cool_achievement") -- grant or revoke achievements as you wish
+achievements.advance("my_progress_achievement", 4) -- add 4 to your achievement's progress, granting automagically if progress completes
+achievements.save() -- write your achievements to disk
+```
+
+## Configuring Achievements
+The most important part of configuring your achievements is properly constructing the achievement data blob and calling `achievements.initialize()` to properly set your game up. Calling `.initialize()` during your game's setup will ensure that the library is ready for use from within your game. It will not overwrite or reset previously saved achievement data.
+
+### Example
+```lua
+local data = {
+    -- This example infers most of the game metadata from your game's pdxinfo.
+    iconPath = "assets/achievements/game_icon", -- Update these paths to match your game's file structure. See below for more details.
+    cardPath = "assets/achievements/game_card", 
+    achievements = {
+        {
+            id = "my_achievement_1",
+            name = "Achievement 1",
+            description = "Achievement 1 Description",
+            scoreValue = 3 -- this achievement is 3x more important to game completion as other achievements

```

Comment:
Can you make the first one just id, name, and description? Since those are the 3 minimum pieces you need for an achievement. (It would be good to note that too.)